docs/docs/governance/unlock-dao-tokens.mdx

```diff
@@ -119,3 +119,91 @@ At that point, the collected fees can be "swapped and burned" by calling the `sw
 ### Swapping UDT for UP
 
 If you have earn UDT on any network, you can swap them for UP. In order to do so, you first need to "bring" the UDT back to Base (using a bridge like [Superbridge](https://superbridge.app/base)), and then swap them for UP. The migration is a 1:1000 swap and can be conveniently achieved through a [migration website](https://up-swap.unlock-protocol.com/).
+
+## What is the Unlock Protocol UP token (UP)?
```

Comment:
Let's maybe add a title (## Frequently Asked Questions) above these and "downgrade" these to h3 (###)

Reply:
FAQ title and section added